### ISSUE TITLE
After installation, git name and e-mail changed

### ISSUE THREAD

**Issue by kirushyk** [paulmillr/dotfiles#24]
I accidentally pushed several commit as Paul Miller to our private repo :-(

**Comment by paulmillr** [comment]
Yeah, that's expected - I don't see any other way. There is a README instruction to change your name / email. Basically `.gitconfig` is committed in.

To change commits, use `git push —force`
